explainability-core/src/test/java/org/kie/trustyai/explainability/local/counterfactual/CounterfactualDiversifierTest.java

```diff
@@ -92,6 +104,9 @@ void simpleDiverse(int seed) throws ExecutionException, InterruptedException, Ti
         }
     }
 
+    // these tests fail to find diverse counterfactuals; the original CF is too sparse + close to decision
+    // boundary for the diversifier to work
+    @Disabled
```

Comment:
Please add ticket number
```suggestion
    @Disabled("TICKET")
```

Reply:
done